common-libs/url-resolver/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultTemplate, predefinedTemplates } from './templates'
+import { predefinedUrls } from './urls'
+import { Service } from './services'
+
+function resolveUrl(service: Service, env: string, userTemplate?: string): string {
```

Comment:
`env` should be of type `'dev' | 'staging' | 'prod'` (or you can make it an enum if you want to)